stsynphot/spectrum.py

```diff
@@ -143,8 +143,12 @@ def interpolate_spectral_element(parfilename, interpval, ext=1):
     col_prefix = xre.group('col').upper()
 
     # Read data table
-    data, wave_unit, doshift, extrapolate = stio.read_interp_spec(
-        filename, tab_ext=ext)
+    try:
+        data, wave_unit, doshift, extrapolate = stio.read_interp_spec(
+            filename, tab_ext=ext)
+    except Exception as e:
```

Comment:
If everything goes well, we won't encounter this.

```suggestion
    except Exception as e:  # pragma: no cover
```